include/pfs/drp/stella/SpectraTemplates.hpp

```diff
@@ -532,10 +529,16 @@ bool pfs::drp::stella::Spectrum<SpectrumT, MaskT, VarianceT, WavelengthT>::ident
     #endif
     
     /// Remove lines rejected by PolyFit from D_A1_FittedPos and D_A1_FittedWLen
-    #ifdef __DEBUG_IDENTIFY__
-        for (int i = 0; i < rejected->size(); ++i)
-            cout << "identify: rejected[" << i << "] = " << (*rejected)[i] << endl;
-    #endif
+    _mask.addMaskPlane("REJECTED_LINES");
```

Comment:
Where should that go then? Into the constructor of Spectrum? Or should I add another parameter to Spectrum::identify?